projects/smoke_bts7200/src/current_measurement_config.c

```diff
@@ -0,0 +1,212 @@
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_SOLAR_SENSE,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_TELEMETRY,
+              .mux_selection = 7,
+          },
+          {
+              .dsel_pin = REAR_PIN_STROBE_CTR_BRK_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_STROBE,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_CENTRE_BRAKE_LIGHT,
+              .mux_selection = 9,
+          },
+          {
+              .dsel_pin = REAR_PIN_REAR_TURN_LIGHT_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_LEFT_REAR_TURN_LIGHT,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_RIGHT_REAR_TURN_LIGHT,
+              .mux_selection = 0,
+          },
+          {
+              .dsel_pin = REAR_PIN_REAR_BRAKE_LIGHT_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_LEFT_BRAKE_LIGHT,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_RIGHT_BRAKE_LIGHT,
+              .mux_selection = 4,
+          },
+          {
+              .dsel_pin = REAR_PIN_CAM_SPARE_10_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_REAR_CAMERA,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_10,
+              .mux_selection = 12,
+          },
+          {
+              .dsel_pin = REAR_PIN_SPARE_1_2_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_1,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_2,
+              .mux_selection = 1,
+          },
+          {
+              .dsel_pin = REAR_PIN_SPARE_3_4_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_3,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_4,
+              .mux_selection = 2,
+          },
+          {
+              .dsel_pin = REAR_PIN_SPARE_8_9_DSEL,
+              .current_0 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_8,
+              .current_1 = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_9,
+              .mux_selection = 6,
+          },
+      },
+  .num_bts7200_channels = 8,
+  .bts7040s =
+      (PowerDistributionBts7040Data[]){
+          {
+              .current = REAR_POWER_DISTRIBUTION_CURRENT_BMS_CARRIER,
+              .mux_selection = 8,
+          },
+          {
+              .current = REAR_POWER_DISTRIBUTION_CURRENT_MCI,
+              .mux_selection = 13,
+          },
+          {
+              .current = REAR_POWER_DISTRIBUTION_CURRENT_CHARGER,
+              .mux_selection = 11,
+          },
+          {
+              .current = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_5,
+              .mux_selection = 10,
+          },
+          {
+              .current = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_6,
+              .mux_selection = 3,
+          },
+          {
+              .current = REAR_POWER_DISTRIBUTION_CURRENT_SPARE_7,
+              .mux_selection = 5,
+          },
+      },
+  .num_bts7040_channels = 6,
```

Comment:
Since this isn't a BTS7040 smoke test, you can remove this since you aren't using it.